src/Events/QR/Controller.php

```diff
@@ -104,7 +118,19 @@ public function add_valid_pages( $valid_pages ) {
 	 * @return void
 	 */
 	protected function register_assets(): void {
-		// @TODO load our QR CSS and JS here using TEC\Common\StellarWP\Asset
+		Asset::add(
+			'tec-events-qr-code-styles',
+			TEC::instance()->plugin_url . 'src/resources/css/qr-code.css'
+		)
+			->enqueue_on( 'admin_enqueue_scripts' )
+			->register();
+
+		Asset::add(
+			'tec-events-qr-code-scripts',
+			TEC::instance()->plugin_url . 'src/resources/js/qr-code.js'
+		)
+			->enqueue_on( 'admin_enqueue_scripts' )
+			->register();
```

Comment:
This is not the correct way to enqueue assets - Path should be provided NOT URL!

URL will work but you are missing 2 crucial features that is provided by the assets:

1) auto-figuring out if there is a `.min` of the asset
2) auto-figuring out if there is a dependency file for the asset

Reply:
Also provide a conditional - these two will be enqueued for all the admin. Thats the desired or only specific page(s) ?